nbsite/tmplate2/conf.py

```diff
@@ -36,15 +38,15 @@
     'PROJECT': project,
     'DESCRIPTION': description,
     'AUTHOR': authors,
-    # will work without this - for canonical (so can ignore when building locally or test deploying)    
-    'WEBSITE_SERVER': 'https://ceball.github.io',
+    # will work without this - for canonical (so can ignore when building locally or test deploying)
+    'WEBSITE_SERVER': '//{}.pyviz.org'.format(project),
     'VERSION': version,
     'NAV': _NAV,
     # by default, footer links are same as those in header
     'LINKS': _NAV,
     'SOCIAL': (
-        ('Gitter', '//gitter.im/ioam/holoviews'),
-        ('Twitter', '//twitter.com/holoviews'),
-        ('Github', '//github.com/ioam/holoviews'),
+        ('Gitter', '//gitter.im/pyviz/pyviz'),
+        ('Twitter', '//twitter.com/pyviz_org'),
+        ('Github', '//github.com/pyviz/{}'.format(project)),
```

Comment:
I don't think this link format will work when the website is used locally, but I guess that's rare?